src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-last-child(n + 5),
+.stars--2 > .stars__star:nth-last-child(n + 4),
+.stars--3 > .stars__star:nth-last-child(n + 3),
+.stars--4 > .stars__star:nth-last-child(n + 2),
+.stars--5 > .stars__star:nth-last-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```

Try to use `:nth-child` and get rid of the increasing in specificity
```suggestion
.stars--1:nth-child(****),
.stars--2:nth-child(****),
.stars--3:nth-child(****),
.stars--4:nth-child(****),
.stars--5:nth-child(****) {
  background-image: url(./images/star-active.svg);
}
```